src/test/CliptoExchange.t.sol

```diff
@@ -15,10 +15,15 @@ contract CliptoExchangeTest is DSTestPlus, IERC721Receiver {
 
     function testCreatorRegistration() public {
         // Register creator.
-        address tokenAddress = exchange.registerCreator("Gabriel", "https://arweave.net/0xprofileurl", 1e18);
+        exchange.registerCreator(
+            "Gabriel", 
+            "https://arweave.net/0xprofileurl", 
+            1e18
+        );
 
         // Retrieve creator information.
         (string memory profileUrl, uint256 cost, CliptoToken token) = exchange.creators(address(this));
+        address tokenAddress = address(token);
```

Comment:
I think this means we're basically doing an `assertEq` on the same variable lower down. We can just remove it I think.